_data/blogs.yml

```diff
@@ -251,4 +251,10 @@
   title: Interviewing Contributors !!
   img: Kothari.jpg
   author_name: Yash Kothari
+  
+- url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title: Here I come Coding In
+  img: https://user-images.githubusercontent.com/34957702/34479624-e24cb40c-efcd-11e7-83f4-7921fe396574.JPG
```

Comment:
Dont paste the image url here. Put the image name after uploading to img/blog/

Reply:
But uploads are diasabled